tests/integration_tests/batch_scripts/summit.sh

```diff
@@ -62,18 +62,18 @@ if test "$?" = 1
 then
     echo 'Test failed'
     tests/bin/radical-pilot-test-issue -r 'ORNL Summit' -l output.log
-    time=`date +'%Y:%m:%d:%H:%M:%S' -d 'now + 1 month'`
-    bsub -b=$time summit.sh
     curl -H "Accept: application/vnd.github.everest-preview+json" \
     -H "Authorization: token $GIT_TOKEN" \
     --request POST \
     --data '{"event_type": "test_summit", "client_payload": { "text": "failure"}}' \
     https://api.github.com/repos/radical-cybertools/radical.pilot/dispatches
+    time=`date +'%Y:%m:%d:%H:%M' -d 'now + 1 month'`
+    bsub -b $time summit.sh
 
 else
     echo 'Everything went well'
-    time=`date + '%Y:%m:%d:%H:%M:%S' -d 'now + 1 week'`
-    bsub -b=$time summit.sh
+    time=`date +'%Y:%m:%d:%H:%M' -d 'now + 1 week'`
+    bsub -b $time summit.sh
```

Comment:
@iparask sorry, I got a little confused, you moved `bsub` after `curl` command for failed test, is it important so it would executed after? and it is not a case for success test?

Reply:
IIUC, this is not the submission of *this* test, but of the *next* test,  so I don't think the order matters.

Reply:
Yes, it is the submission of the next test and it does not matter.